apps/info.c

```diff
@@ -30,6 +30,7 @@ const OPTIONS info_options[] = {
     {"dsoext", OPT_DSOEXT, '-', "Configured extension for modules"},
     {"dirnamesep", OPT_DIRNAMESEP, '-', "Directory-filename separator"},
     {"listsep", OPT_LISTSEP, '-', "List separator character"},
+    {"seedsources", OPT_SEEDSOURCES, '-', "Seed sources"},
```

Comment:
What about `seeds` instead?  A bit easier to type :)

Reply:
Okie